<ui version="4.0" >
 <class>MitesControls</class>
 <widget class="QWidget" name="MitesControls" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Controls</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6" >
     <item>
      <widget class="QLabel" name="label_5" >
       <property name="text" >
        <string>Zoom</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="ZoomSlider" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Mites</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="MitesSpinBox" >
       <property name="maximum" >
        <number>200</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSlider" name="MitesSlider" >
     <property name="maximum" >
      <number>200</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <spacer name="horizontalSpacer_2" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Chips</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="ChipsSpinBox" >
       <property name="maximum" >
        <number>200</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSlider" name="ChipsSlider" >
     <property name="maximum" >
      <number>200</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" >
     <item>
      <spacer name="horizontalSpacer_3" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Speed</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="SpeedSpinBox" >
       <property name="maximum" >
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSlider" name="SpeedSlider" >
     <property name="maximum" >
      <number>100</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7" >
     <item>
      <spacer name="horizontalSpacer_7" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_6" >
       <property name="text" >
        <string>Max Turn</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="TurnSpinBox" >
       <property name="minimum" >
        <number>1</number>
       </property>
       <property name="maximum" >
        <number>360</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSlider" name="TurnSlider" >
     <property name="minimum" >
      <number>1</number>
     </property>
     <property name="maximum" >
      <number>360</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8" >
     <item>
      <spacer name="horizontalSpacer_8" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_7" >
       <property name="text" >
        <string>Turn Delay</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="TurnDelaySpinBox" >
       <property name="minimum" >
        <number>0</number>
       </property>
       <property name="maximum" >
        <number>100</number>
       </property>
       <property name="value" >
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSlider" name="TurnDelaySlider" >
     <property name="minimum" >
      <number>0</number>
     </property>
     <property name="maximum" >
      <number>100</number>
     </property>
     <property name="sliderPosition" >
      <number>0</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" >
     <item>
      <spacer name="horizontalSpacer_4" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>Hunt Delay</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="WaitSpinBox" >
       <property name="minimum" >
        <number>1</number>
       </property>
       <property name="maximum" >
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSlider" name="WaitSlider" >
     <property name="minimum" >
      <number>1</number>
     </property>
     <property name="maximum" >
      <number>100</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>17</width>
       <height>242</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" >
     <item>
      <widget class="QPushButton" name="PauseButton" >
       <property name="text" >
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>118</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ResetButton" >
       <property name="text" >
        <string>Reset</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ZoomSlider</tabstop>
  <tabstop>MitesSpinBox</tabstop>
  <tabstop>MitesSlider</tabstop>
  <tabstop>ChipsSpinBox</tabstop>
  <tabstop>ChipsSlider</tabstop>
  <tabstop>SpeedSpinBox</tabstop>
  <tabstop>SpeedSlider</tabstop>
  <tabstop>TurnSpinBox</tabstop>
  <tabstop>TurnSlider</tabstop>
  <tabstop>TurnDelaySpinBox</tabstop>
  <tabstop>TurnDelaySlider</tabstop>
  <tabstop>WaitSpinBox</tabstop>
  <tabstop>WaitSlider</tabstop>
  <tabstop>PauseButton</tabstop>
  <tabstop>ResetButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>MitesSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MitesSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>96</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel" >
     <x>116</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MitesSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MitesSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>116</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel" >
     <x>72</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ChipsSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ChipsSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>118</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel" >
     <x>84</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ChipsSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ChipsSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>117</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel" >
     <x>118</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SpeedSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SpeedSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>119</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel" >
     <x>73</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SpeedSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SpeedSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>143</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel" >
     <x>119</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>WaitSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>WaitSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>117</x>
     <y>313</y>
    </hint>
    <hint type="destinationlabel" >
     <x>108</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>WaitSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>WaitSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>169</x>
     <y>348</y>
    </hint>
    <hint type="destinationlabel" >
     <x>117</x>
     <y>313</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TurnSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>TurnSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>75</x>
     <y>408</y>
    </hint>
    <hint type="destinationlabel" >
     <x>90</x>
     <y>372</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TurnDelaySlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>TurnDelaySpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>64</x>
     <y>483</y>
    </hint>
    <hint type="destinationlabel" >
     <x>119</x>
     <y>461</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TurnDelaySpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>TurnDelaySlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>140</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel" >
     <x>241</x>
     <y>481</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TurnSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>TurnSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>139</x>
     <y>386</y>
    </hint>
    <hint type="destinationlabel" >
     <x>240</x>
     <y>404</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
